contracts/Claims.sol

```diff
@@ -223,9 +223,16 @@ contract Claims is AccessControlEnumerable {
     // various boolean flags used here, see VOTE_STAKE_Y, VOTE_STAKE_N,
     // VOTE_TRUTH_Y, VOTE_TRUTH_N, VOTE_TRUTH_S and VOTE_TRUTH_V.
     mapping(uint256 => mapping(address => Bits.Map)) private _addressVotes;
-    //
+    // _allocBalance contains the allocated balance every user has in this smart
+    // contract. Allocated balances may increase by creating claims or by
+    // participating in their markets through staking. The allocated balance is
+    // the part of the user balance that cannot be withdrawn.
     mapping(address => uint256) private _allocBalance;
-    // TODO put together in one mapping with _allocBalance
```

Comment:
That was an idea, which, to my surprise, turned out to be more gas consuming. So we just keep it as it is and move on.